dwave/preprocessing/include/dwave/presolve.h

```diff
@@ -415,6 +424,48 @@ class Presolver {
 
         return empty_interactions.size() || empty_variables.size();
     }
+
+    static bool remove_small_biases(dimod::Constraint<bias_type, index_type>& expression) {
+        if (expression.is_linear()) {
```

Comment:
The logic looks fine, but IMO the flow would be a lot clearer with
```c++
// Not yet defined for quadratic expressions (TODO)
if (!expression.is_linear()) return false;
```
and the rest of the function outside of an `if` branch.